examples/simple_hpc_pbs_singularity/config.json

```diff
@@ -0,0 +1,132 @@
+{
+    "location": "variables.location",
+    "resource_group": "variables.resource_group",
+    "install_from": "headnode",
+    "admin_user": "hpcadmin",
+    "variables": {
+        "acr_repo": "<NOT-SET>",
+        "key_vault": "<NOT-SET>",
+        "monitor_workspace": "<NOT-SET>",
+        "hpc_image": "OpenLogic:CentOS-HPC:7.6:latest",
+        "location": "southcentralus",
+        "resource_group": "<NOT-SET>",
+        "vm_type": "Standard_HB60rs",
```

Comment:
Please don't use HB for Headnode, use DS_v3 instead. Add an additional VM type variable for the headnode as well as and additional image.

Reply:
Changed.